<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenshinImpact_TianLiClass</class>
 <widget class="QMainWindow" name="GenshinImpact_TianLiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1573</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GenshinImpact_TianLi.qrc">
    <normaloff>:/Logo/resource/Icon.png</normaloff>:/Logo/resource/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" rowspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1550</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1550</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label_Main">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>1521</width>
         <height>841</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
background-image:url(:/GenshinImpact_TianLi/resource/主界面背景.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Exit">
       <property name="geometry">
        <rect>
         <x>1460</x>
         <y>100</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
border:none;
image:url(:/GenshinImpact_TianLi/resource/退出按钮.png);

}
.QPushButton:hover{
border:none;
image:url(:/GenshinImpact_TianLi/resource/退出按钮_变暗.png);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_MainPage">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>1291</width>
         <height>771</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">.QLabel
{
background-image:url(:/GenshinImpact_TianLi/resource/书册.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Tab_1">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>74</y>
         <width>165</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>165</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>165</width>
         <height>91</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>HYWenHei</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
border:none;
background-repeat:no-repeat;
background-origin:content;
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签一.png);

padding-left: 9px;
padding-right: 21px;
font-size:25px;
color:rgb(226, 210, 191);
}
.QPushButton:hover{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签一_变亮.png);

padding-left: 9px;

padding-right: 21px;

}
.QPushButton:checked{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/Tab_地图_selected.png);

padding-left: 0px;

padding-right: -9px;
color:rgb(134, 99, 67);
}</string>
       </property>
       <property name="text">
        <string>地图</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Tab_ButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QPushButton" name="pushButton_Tab_2">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>175</y>
         <width>165</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HYWenHei</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
border:none;
background-repeat:no-repeat;
background-origin:content;
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签二.png);

padding-left: 9px;
padding-right: 21px;
font-size:25px;
color:rgb(226, 210, 191);
}
.QPushButton:hover{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签二_变亮.png);

padding-left: 9px;

padding-right: 21px;

}
.QPushButton:checked{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/Tab_联机_selected.png);

padding-left: 0px;

padding-right: -9px;
color:rgb(134, 99, 67);
}</string>
       </property>
       <property name="text">
        <string>界面</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Tab_ButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QPushButton" name="pushButton_Tab_3">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>276</y>
         <width>165</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HYWenHei</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
border:none;
background-repeat:no-repeat;
background-origin:content;
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签三.png);

padding-left: 9px;
padding-right: 21px;
font-size:25px;
color:rgb(226, 210, 191);
}
.QPushButton:hover{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签三_变亮.png);

padding-left: 9px;

padding-right: 21px;

}
.QPushButton:checked{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/Tab_联机_selected.png);

padding-left: 0px;

padding-right: -9px;
color:rgb(134, 99, 67);
}</string>
       </property>
       <property name="text">
        <string>联机</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Tab_ButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QPushButton" name="pushButton_Tab_4">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>377</y>
         <width>165</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>HYWenHei</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton{
border:none;
background-repeat:no-repeat;
background-origin:content;
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签四.png);

padding-left: 9px;
padding-right: 21px;
font-size:25px;
color:rgb(226, 210, 191);
}
.QPushButton:hover{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/冒险手册_标签四_变亮.png);

padding-left: 9px;

padding-right: 21px;

}
.QPushButton:checked{
background-image:url(:/TianLi_Tab/resource/TianLi_Tab/Tab_特殊_selected.png);

padding-left: 0px;

padding-right: -9px;
color:rgb(134, 99, 67);
}</string>
       </property>
       <property name="text">
        <string>特殊</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">Tab_ButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget_MainTabPages">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>1271</width>
         <height>751</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_MapTab_Left" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_MapTab_Right" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="widget_MapMask" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>621</width>
              <height>731</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_MapMask">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="widget_HUDTab" native="true">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>6</y>
           <width>1259</width>
           <height>739</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3"/>
       <widget class="QWidget" name="page_4"/>
       <widget class="QWidget" name="page_set"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>74</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_Set">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>452</y>
         <width>89</width>
         <height>278</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton
{
border:none;
background-image: url(:/GenshinImpact_TianLi/resource/冒险手册_钥匙.png);
}
.QPushButton:hover
{
border:none;
background-image: url(:/GenshinImpact_TianLi/resource/冒险手册_钥匙.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_Main</zorder>
      <zorder>pushButton_Set</zorder>
      <zorder>pushButton_Exit</zorder>
      <zorder>label_MainPage</zorder>
      <zorder>stackedWidget_MainTabPages</zorder>
      <zorder>verticalLayoutWidget</zorder>
      <zorder>pushButton_Tab_1</zorder>
      <zorder>pushButton_Tab_2</zorder>
      <zorder>pushButton_Tab_3</zorder>
      <zorder>pushButton_Tab_4</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_Tab_1</tabstop>
  <tabstop>pushButton_Tab_2</tabstop>
  <tabstop>pushButton_Tab_3</tabstop>
  <tabstop>pushButton_Tab_4</tabstop>
  <tabstop>pushButton_Exit</tabstop>
 </tabstops>
 <resources>
  <include location="GenshinImpact_TianLi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>GenshinImpact_TianLiClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1470</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="Tab_ButtonGroup"/>
 </buttongroups>
</ui>
